conmon/conmon.c

```diff
@@ -825,13 +851,16 @@ static gboolean ctrl_cb(int fd, G_GNUC_UNUSED GIOCondition condition, G_GNUC_UNU
 			return G_SOURCE_CONTINUE;
 		}
 		ninfo("Message type: %d, Height: %d, Width: %d", ctl_msg_type, height, width);
```

Comment:
Is this (and the above lines) really the format for each line?  I can understand `1 {height} {width}\n` for resize requests, but `2 {dummy} {dummy}\n" seems strange for log-reopen requests.

Reply:
Not worth the effort to support multiple size of the control messages IMO. We can always clean it up later.